backend/src/helpers/auth.ts

```diff
@@ -382,11 +460,15 @@ export const createToken = ({
 	secret,
 }: {
 	payload: any;
-	expiresIn: string | number;
+	expiresIn?: string | number;
 	secret: string;
 }) => {
 	return jwt.sign(payload, secret, {
-		expiresIn,
+		...(
+			(expiresIn !== undefined && expiresIn !== null) 
```

Comment:
nit: you can do short hand `expiresIn != null`

Reply:
It can still be undefined which is different from null.